render/wgpu/src/surface.rs

```diff
@@ -228,8 +231,13 @@ impl Surface {
                     let parent = match blend_mode {
                         ComplexBlend::Alpha | ComplexBlend::Erase => match nearest_layer {
                             LayerRef::None => {
-                                // An Alpha or Erase with no Layer above it should be ignored
-                                continue;
+                                if render_mode_existing_texture {
```

Comment:
If I understand this right, this makes `LayerRef::None` act like `LayerRef::Current` for the deepest layer. Is that right? If there's things between, I somewhat expected it to become the the furthest ancestor?

Could you check that please? I think just passing in `LayerRef::Current` in the initial call to `draw_commands` for the case of bmd.draw() would do that behaviour, but we need to see which one is right first

Reply:
@Dinnerbone It looks like we only ever create `LayerRef::None` at the top level `draw_commands_to` (called from `render_offscreen` and `submit_frame`), so we'll never have any ancestor layers when we see it. However, I think you're right that it makes more sense to explicitly pass in `LayerRef::Current`.